ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/_cq_dialog/.content.xml

```diff
@@ -72,6 +72,88 @@
+                                            <optionNone
+                                                    jcr:primaryType="nt:unstructured"
+                                                    text="None"
+                                                    value="none"/>
+                                            <optionSchema
+                                                    jcr:primaryType="nt:unstructured"
+                                                    text="Schema"
+                                                    value="schema"/>
+                                            <optionFormdatamodel
+                                                    jcr:primaryType="nt:unstructured"
+                                                    text="Form Data Model"
+                                                    value="formdatamodel"/>
+                                        </items>
+                                    </selectformmodel>
+                                    <schemaRef name="./schemaRef"
+                                               jcr:primaryType="nt:unstructured"
+                                               sling:resourceType="granite/ui/components/coral/foundation/form/hidden"/>
+                                    <schemaSelectorContainer
+                                            jcr:primaryType="nt:unstructured"
+                                            sling:resourceType="granite/ui/components/coral/foundation/container"
+                                            granite:class="cmp-adaptiveform-container__schemaselectorcontainer">
+                                        <items jcr:primaryType="nt:unstructured">
+                                            <schemaSelector
+                                                    granite:class="cmp-adaptiveform-container__schemaselector"
+                                                    jcr:primaryType="nt:unstructured"
+                                                    sling:resourceType="granite/ui/components/coral/foundation/form/select"
+                                                    disabled="false"
+                                                    fieldLabel="Select Schema *"
+                                                    validation="datamodel.config"
+                                                    renderReadOnly="false">
+                                                <datasource
+                                                        jcr:primaryType="nt:unstructured"
+                                                        sling:resourceType="fd/af/components/commons/datasources/propertyprovider"
+                                                        guideDataModel="basic"
+                                                        layoutType="fd/af/layouts/panel"
+                                                        type="xmljsonschema"/>
```

Comment:
what is `xmljsonschema`?

Reply:
existing identifier to fetch all the schemas from property provider